scripts/install-linux-deps.sh

```diff
@@ -48,13 +48,13 @@ env DEBIAN_FRONTEND=noninteractive apt-get install --yes --quiet \
   
 # build prime_server from source
 # readonly primeserver_version=0.7.0
-readonly primeserver_dir=/usr/local/src/prime_server
+readonly primeserver_dir=/tmp/prime_server
```

Comment:
on the regular linux builds we no longer run this script as sudo, instead using sudo inside of the script only where its needed. because of that `/tmp` is a better place for temporary stuff